Build/SaveLoadScreen.cc

```diff
@@ -1580,8 +1830,16 @@ static void SaveGameToSlotNum(void)
 	//render the buttons
 	MarkButtonsDirty( );
 	RenderButtons();
-
-	if( !SaveGame( gbSelectedSaveLocation, gzGameDescTextField ) )
+  
+  // If we are selecting the Dead is Dead Savegame slot, only remember the slot, do not save
+  // Also set the INTRO_SCREEN as previous options screen. This is a hack to get the game started
+  if (guiPreviousOptionScreen == GAME_INIT_OPTIONS_SCREEN)
+  {
+    guiPreviousOptionScreen = INTRO_SCREEN;
+    gGameSettings.bLastSavedGameSlot = (gbSelectedSaveLocation + NUM_SAVE_GAMES);
+    wcscpy(gGameSettings.sCurrentSavedGameName, gzGameDescTextField);
```

Comment:
It is.
The condition is true only if we switch from the Game Init Options Screen to the Save screen.
This is only possible with the new game mode selected in the Game Init Options Screen.
There is no other way of accessing the Save Screen from within the GIO Screen, and guiPreviousOptionScreen can never be GAME_INIT_OPTIONS_SCREEN in any other instance.

This was the most elegant solution I could find since the GameSettings cannot yet be read at this point.
Also tried a lot and could not get it to fail.